tamr_unify_client/project/attribute_mapping/resource.py

```diff
@@ -0,0 +1,80 @@
+class AttributeMapping:
+    """see https://docs.tamr.com/reference#retrieve-projects-mappings
+       AttributeMapping and AttributeMappingCollection do not inherit from BaseResource and BaseCollection.
+        BC and BR require a specific URL for each individual attribute mapping
+        (ex: /projects/1/attributeMappings/1), but these types of URLs do not exist for attribute mappings
+    """
+
+    def __init__(self, data):
+        self._data = data
+
+    @property
+    def id(self):
+        """:type: str"""
+        return self._data["id"]
+
+    @property
+    def relative_id(self):
+        """:type: str"""
+        return self._data["relativeId"]
+
+    @property
+    def input_attribute_id(self):
+        """:type: str"""
+        return self._data["inputAttributeId"]
+
+    @property
+    def relative_input_attribute_id(self):
+        """:type: str"""
+        return self._data["relativeInputAttributeId"]
+
+    @property
+    def input_dataset_name(self):
+        """:type: str"""
+        return self._data["inputDatasetName"]
+
+    @property
+    def input_attribute_name(self):
+        """:type: str"""
+        return self._data["inputAttributeName"]
+
+    @property
+    def unified_attribute_id(self):
+        """:type: str"""
+        return self._data["unifiedAttributeId"]
+
+    @property
+    def relative_unified_attribute_id(self):
+        """:type: str"""
+        return self._data["relativeUnifiedAttributeId"]
+
+    @property
+    def unified_dataset_name(self):
+        """:type: str"""
+        return self._data["unifiedDatasetName"]
+
+    @property
+    def unified_attribute_name(self):
+        """:type: str"""
+        return self._data["unifiedAttributeName"]
+
+    def resource_id(self, input_relative):
```

Comment:
What's the reasoning for this not to be a property that works with `self.relative_id`?